docs/modules/ROOT/pages/ec_test.adoc

```diff
@@ -83,7 +83,7 @@ the output will include a detailed trace of how the policy was evaluated, e.g.
 -n, --namespace:: Test policies in a specific namespace (Default: [main])
 --no-color:: Disable color when printing (Default: false)
 --no-fail:: Return an exit code of zero even if a policy fails (Default: false)
--o, --output:: Output format for conftest results - valid options are: [stdout json tap table junit github appstudio] (Default: stdout)
+-o, --output:: Output format for conftest results - valid options are: [stdout json tap table junit github appstudio] (Default: [])
```

Comment:
I know it's generated, but the "Default" here is not helpful. Can you think of a way to have it still say stdout?

Also, I think it would be good if the help text somehow mention the `format=filename` option. Maybe add a little extra on line 268.

Reply:
@simonbaird I don't think there's a way to specify a default value with `StringSliceP`, so we may just need to live with the empty default.

We could also manually call it out in the help text, e.g. "defaults to stdout" or something.